bundles/org.openhab.ui.basic/web-src/_layout.scss

```diff
@@ -232,6 +232,11 @@
 			font-size: 18px;
 		}
 	}
+	&__label:has(+ .mdl-form__text) {
+		// this applies to labels that are followed by a text input
```

Comment:
Not "text input".
Maybe rather something like "this applies only to labels for sitemap text element" ?

Reply:
comment fixed. thanks for catching it.